transmart-rest-api/grails-app/controllers/org/transmartproject/rest/ExportController.groovy

```diff
@@ -257,4 +245,14 @@ class ExportController {
                 componentType: Map,
         )
     }
+
+    private Constraint getConstraintFromJson(json, User user) {
```

Comment:
This doesn't seem right.

Reply:
@gijskant  Could you please tell more? What would be right?

Reply:
We will change later.